### PR TITLE
Admin Screen Nav

### PR DESCRIPTION
**Pull request by mbeargie1238**
Set up bottom nav for admin screen - added screen for user management (add/remove users - change permissions) - added screen for scheduling a livestream.

Left library screen as is on admin nav. I wasn't sure if that was potentially a screen where recorded streams could be edited by an admin.

First time working with Dart. Any and all feedback welcome. Thanks!

### CODE REVIEW

File: lib/views/user_management/user_management.dart
Comment (nate-thegrate) [inline lib/views/user_management/user_management.dart:1]: Now that we have screens for user management and scheduling livestreams, I don't think the `admin_portal` folder is needed anymore. Do you think you could delete that one?

File: lib/views/home/admin_home.dart [hunk 1]
```diff
@@ -1,11 +1,119 @@
...
+    return Scaffold(
+      bottomNavigationBar: navigationBar,
+      body: switch (navigationBar.selectedIndex) {
+        0 => const WatchLive(),
+        1 => const ScheduleLivestream(),
+        2 => const VideoLibrary(),
+        3 => const UserManagement(),
+        _ => const SettingsScreen(),
+      },
+    );
```
Comment (nate-thegrate) [inline lib/views/home/admin_home.dart:26]: Fantastic work here 👍

**Review by nate-thegrate — COMMENTED** [review]
Thanks for setting this up!

File: lib/views/user_management/user_management.dart
Comment (mbeargie1238) [reply lib/views/user_management/user_management.dart:1]: Thanks for reviewing! Good call I'll update this PR here in a bit, thanks!

**Review by nate-thegrate — COMMENTED** [review]
Thank you!

Let's hold off on merging for a little bit if you don't mind—I'd like to mess around with the class structure and see if a few things can be consolidated.

File: lib/views/user_management/user_management.dart
Comment (mbeargie1238) [reply lib/views/user_management/user_management.dart:1]: `admin_portal` folder should be removed now. Any issues let me know, thanks!

**Review by nate-thegrate — APPROVED** [review]
Since the Director & Admin home screens are so similar, I thought it'd be nice to merge the respective classes.

Thanks again for this contribution!

**Pull request merged**